src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalyticsSDK.scala

```diff
@@ -1,20 +1,26 @@
 package com.microsoft.ml.spark.cognitive
-import com.azure.ai.textanalytics.models.{AssessmentSentiment, DetectLanguageInput, DocumentSentiment,
-  SentenceSentiment, SentimentConfidenceScores, TargetSentiment, TextDocumentInput}
+import com.azure.ai.textanalytics.models.{AssessmentSentiment, DetectLanguageInput, DocumentSentiment, SentenceSentiment, SentimentConfidenceScores, TargetSentiment, TextDocumentInput}
 import com.azure.ai.textanalytics.{TextAnalyticsClient, TextAnalyticsClientBuilder}
 import com.azure.core.credential.AzureKeyCredential
-import com.microsoft.ml.spark.core.contracts.{HasConfidenceScoreCol, HasInputCol, HasOutputCol, HasTextCol, HasLangCol}
-import com.microsoft.ml.spark.core.schema.{SparkBindings}
-import com.microsoft.ml.spark.io.http.{HasErrorCol}
+import com.microsoft.ml.spark.core.contracts.{HasConfidenceScoreCol, HasInputCol, HasLangCol, HasOutputCol, HasTextCol}
+import com.microsoft.ml.spark.core.schema.SparkBindings
+import com.microsoft.ml.spark.io.http.HasErrorCol
 import com.microsoft.ml.spark.logging.BasicLogging
-import org.apache.spark.ml.param.{ParamMap}
+import org.apache.spark.ml.param.{BooleanParam, IntParam, Param, ParamMap, Params}
 import org.apache.spark.ml.util.Identifiable._
 import org.apache.spark.ml.{ComplexParamsReadable, ComplexParamsWritable, Transformer}
-import org.apache.spark.sql.types.{DataTypes, StructType}
+import org.apache.spark.sql.types.{ArrayType, DataTypes, StringType, StructType}
 import org.apache.spark.sql.{DataFrame, Dataset, Row}
 import com.azure.core.util.Context
+import com.microsoft.ml.spark.stages.{FixedBatcher, FixedBufferedBatcher, FixedMiniBatchTransformer, HasBatchSize, MiniBatchBase}
+import com.sun.tools.javac.code.TypeTag
+import org.apache.spark.injections.UDFUtils
+import org.apache.spark.ml.util.{DefaultParamsReadable, Identifiable}
 import org.apache.spark.sql.catalyst.encoders.RowEncoder
+
+import java.util.concurrent.{BlockingQueue, CountDownLatch, LinkedBlockingQueue}
 import scala.collection.JavaConverters._
+import scala.reflect.runtime.universe.{MethodSymbol, typeOf}

```

Comment:
Is this needed?

Reply:
Ooops forgot to take those out. Were for a different attempt at batching that ended up not being used.